.gitignore

```diff
@@ -37,3 +37,6 @@ yarn-error.*
 
 # webstorm
 .idea/
+
+# env
```

Comment:
No comments

Reply:
not a comment, this is meant as a "category" like the rest 😛